tools/fetch_fasta_from_ncbi/retrieve_fasta_from_NCBI.py

```diff
@@ -166,16 +169,30 @@ def efetch(self, db, query_key, webenv):
             try:
                 response = urllib2.urlopen(req)
                 fasta = response.read()
-                if ("Resource temporarily unavailable" in fasta) or (not fasta.startswith(">") ):
+                if ("Resource temporarily unavailable" in fasta) or ("Error" in fasta) or (not fasta.startswith(">") ):
                     serverTransaction = False
                 else:
                     serverTransaction = True
             except urllib2.HTTPError as e:
+                print "urllib2.HTTPError at ", datetime.now()
                 serverTransaction = False
                 self.logger.info("urlopen error:%s, %s" % (e.code, e.read() ) )
             except httplib.IncompleteRead as e:
+                print "httplib.IncompleteRead at ", datetime.now()
+                serverTransaction = False
+                self.logger.info("IncompleteRead error")
+            except urllib2.URLError as e:
+                print "urllib2.URLError at ", datetime.now()
```

Comment:
I'm unsure how the logger is configured, but it includes the date by default, so unless the logger is silent, the print statements don't add any information.